src/components/TopicCard.jsx

```diff
@@ -0,0 +1,25 @@
+import { Card, CardContent, Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+
+const TopicCard = ({ topic }) => {
+  return (
+    <li key={topic.slug}>
```

Comment:
If you'd like to get rid of the bullet points you can set the ul's css property `list-style-type` to `none`